frontend/src/components/Psychologist/Profile/EditProfile.jsx

```diff
@@ -34,25 +35,49 @@ const EditProfile = ({ psychologist, updatePsy, cancelEditProfile }) => {
         type="email"
         required
       />
-      <SearchableSelect
-        className="midlength-select"
-        label="Votre département"
-        field="departement"
-        data-test-id="psy-departement-select"
-        selected={updatedPsychologist.departement}
-        onChange={e => { changePsychologist(e, 'departement'); }}
-        options={DEPARTEMENTS.map(departement => ({ value: departement, label: departement }))}
-        required
-      />
-      <TextInput
-        className="midlength-input"
-        label="Adresse du cabinet"
-        hint="Adresse où se rendre pour le rendez-vous. Pour indiquer plusieurs adresses, les séparer par un point virgule."
-        data-test-id="psy-address-input"
-        value={updatedPsychologist.address}
-        onChange={e => changePsychologist(e.target.value, 'address')}
-        required
-      />
+      {updatedPsychologist.addresses.map((address, index) => (
+        // eslint-disable-next-line react/no-array-index-key
+        <div key={index} className="fr-mb-3w">
```

Comment:
Why not use `address` as `key` as done in `frontend/src/components/Psychologist/Profile/PsyProfile.jsx`?